bridge-ext/src/main/java/com/liferay/faces/bridge/ext/filter/internal/LiferayURLFactoryImpl.java

```diff
@@ -42,14 +43,15 @@
 		"com.liferay.faces.bridge.container.liferay.RESOURCE_URL_GENERATOR";
 
 	@Override
-	public LiferayActionURL getLiferayActionURL(PortletRequest portletRequest, MimeResponse mimeResponse,
-		boolean friendlyURLMapperEnabled) {
+	public LiferayActionURL getLiferayActionURL(PortletContext portletContext, PortletRequest portletRequest,
```

Comment:
@stiemannkj1 I don't see `portletContext` used anywhere in these method signatures. Please let me know the need for adding this to each of the method signatures since it causes a lot of diffs elsewhere. Thanks.

Reply:
@ngriffin7a, oops, I probably added that when I was obtaining the `FacesURLEncoder` from the `BridgeFactoryFinder`. It's probably not necessary.

Reply:
@stiemannkj1 OK after the merge please send a subsequent PR.